akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -105,7 +105,7 @@ If a message is already enqueued to the entity when it stops itself the enqueued
 in the mailbox will be dropped. To support graceful passivation without losing such
 messages the entity actor can send `ClusterSharding.Passivate` to to the
 @scala[`ActorRef[ShardCommand]`]@java[`ActorRef<ShardCommand>`] that was passed in to
-the factory method when creating the entity. The specified `handOffStopMessage` message
+the factory method when creating the entity. The specified `stopMessage` message
```

Comment:
Optional: "The optional `stopMessage` message will be sent otherwise it will be stopped automatically?

Reply:
fixed